core/backend/src/IModelExporter.ts

```diff
@@ -544,14 +548,14 @@ export class IModelExporter {
       this.handler.callProtected.onExportElement(element, isUpdate);
       this.trackProgress();
       this.exportElementAspects(elementId);
-      this.exportChildElements(elementId);
+      await this.exportChildElements(elementId);

```

Comment:
`return` vs. `await` here for consistency with line 542

Reply:
I made the requested await --> return change.

Based on my conversations with @wgoehrig, I am not expecting any performance hit. However, I will confirm by testing with a large file before I complete this PR.

Reply:
I think there must be some performance impact, even if you return a resolved promise. Would be good to quantify.

Reply:
I will do some direct comparisons of before and after these changes.

Reply:
I have not noticed any adverse performance impacts.

Since the lack of async can cause a "health monitor" to think that the process is hung, I am recommending that these changes be merged in.